tools/abi_downloader/index.ts

```diff
@@ -17,10 +17,12 @@ const allowedABIs = [
 ];
 
 function writeToFile(name, abi, contractVersion): void {
+  const patchVersionIndex = contractVersion.lastIndexOf('.');
+  const majorMinorVersion = contractVersion.slice(0, patchVersionIndex);
```

Comment:
User may not send the patch version `0.10` instead of `0.10.0`

Reply:
If correct version is not passed, npm install will fail. This code will execute only for correct version.
https://github.com/OpenST/mosaic-chains/blob/b0698f9211ddf2a780e01253a8645acc2e657f6e/tools/abi_downloader/run.sh#L10